js/views/RegistrationPages.js

```diff
@@ -123,7 +123,8 @@ const RegistrationPages = React.createClass ({
    */
   beforePageChange(currentPage, nextPage) {
     const shouldContinue = this.validatePage();
-    if (shouldContinue || nextPage < currentPage) {
+    if (shouldContinue || nextPage >= 0 && nextPage < currentPage) {
```

Comment:
That is fine, what that does is allow the registration to move backwards without validating.
It was a bug fix included in #79: A user could not go back on registration to fix things without filling in the current page, which made the swiper confused and soft-locked the view.